Pass in smaller timeout into init_process_group for distributed_test

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#47896 Pass in smaller timeout into init_process_group for distributed_test**

Closes https://github.com/pytorch/pytorch/issues/47892. Since we have a 100s timeout on the entire test, we should have a smaller timeout than the default 30 min for the process group used for the test.

This diff sets the timeout to 60s. For example, this is useful when running tests with NCCL_BLOCKING_WAIT so that we get the op timed out error instead of the test itself timing out. 

Differential Revision: [D24943323](https://our.internmc.facebook.com/intern/diff/D24943323/)

rohan-varma committed:
Pass in smaller timeout into init_process_group for distributed_test

Per title

Differential Revision: [D24943323](https://our.internmc.facebook.com/intern/diff/D24943323/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Pass in smaller timeout into init_process_group for distributed_test"


Per title

Differential Revision: [D24943323](https://our.internmc.facebook.com/intern/diff/D24943323/)

[ghstack-poisoned]

osalpekar reviewed (approved):
Looks like the `fmt` library build error is coming from the other PR in the stack, but this one looks good!

rohan-varma committed:
Update on "Pass in smaller timeout into init_process_group for distributed_test"


Closes https://github.com/pytorch/pytorch/issues/47892. Since we have a 100s timeout on the entire test, we should have a smaller timeout than the default 30 min for the process group used for the test.

This diff sets the timeout to 60s. For example, this is useful when running tests with NCCL_BLOCKING_WAIT so that we get the op timed out error instead of the test itself timing out. 

Differential Revision: [D24943323](https://our.internmc.facebook.com/intern/diff/D24943323/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Pass in smaller timeout into init_process_group for distributed_test"


Closes https://github.com/pytorch/pytorch/issues/47892. Since we have a 100s timeout on the entire test, we should have a smaller timeout than the default 30 min for the process group used for the test.

This diff sets the timeout to 60s. For example, this is useful when running tests with NCCL_BLOCKING_WAIT so that we get the op timed out error instead of the test itself timing out. 

Differential Revision: [D24943323](https://our.internmc.facebook.com/intern/diff/D24943323/)

[ghstack-poisoned]
